modyn/config/schema/pipeline/trigger/drift/alibi_detect.py

```diff
@@ -0,0 +1,57 @@
+from typing import Annotated, Literal, Union
+
+from modyn.config.schema.base_model import ModynBaseModel
+from pydantic import Field
+
+
+class _AlibiDetectBaseDriftMetric(ModynBaseModel):
+    p_val: float = Field(0.05, description="The p-value threshold for the drift detection.")
+
+    # to be added in the future:
+
+    # x_ref_preprocessed
+    # preprocess_at_init
+    # configure_kernel_from_x_ref
```

Comment:
Do we need these for the drift experiments with Modyn? Sorry, not entirely sure what parameters are required.

Reply:
I tried to tweak them, they seem not to have big effects. I have already added them to the interface locally so we can set them if we like. For the paper they won't be of any use though.